app/containers/App/index.js

```diff
@@ -40,7 +40,7 @@ export function App({ location }) {
               <Route
                 exact={routeConfig[routeKey].exact}
                 key={index}
-                path={`${location.pathname}${routeConfig[routeKey].route}`}
+                path={`${routeConfig[routeKey].route}`}
```

Comment:
since we've removed the preceeding location.pathname we don't need a template literal.

```
 path={routeConfig[routeKey].route}
```